avro/src/main/java/com/fasterxml/jackson/dataformat/avro/schema/AvroSchemaGenerator.java

```diff
@@ -31,4 +31,17 @@ public AvroSchemaGenerator disableLogicalTypes() {
         super.disableLogicalTypes();
         return this;
     }
+
+    @Override
+    public AvroSchemaGenerator enableWriteEnumAsString() {
```

Comment:
Quick note: it'd be better to add a new `AvroGenerator.Feature` entry instead of new method(s), if possible.
Would that work?

Reply:
> Yes, AvroGenerator.Feature would be better but I'd need to think of plumbing.
>
> I think AvroGenerator might be available from SerializerProvider but not sure if that is properly initialized.

ATM I do not know how to access `AvroGenerator.Feature` from `VisitorFormatWrapperImpl.expectStringFormat(JavaType)`.

Reply:
I did not find a way to read `AvroFactory`, or how to elegantly pass it into `VisitorFormatWrapperImpl`. :(

Reply:
Ok I hope to have today or tomorrow to see if I can figure out something...

Reply:
@MichalFoksa Answer would `SerializerProvider` which is assigned to `VisitorFormatWrapperImpl` -- it has `getGenerator()` method. Alas, I am not convinced provider gets assigned said generator for this case.

So I'll probably accept this as-is -- let me do final code review and proceed.